Core/include/Acts/Geometry/CylinderVolumeBounds.hpp

```diff
@@ -77,6 +77,7 @@ class CylinderVolumeBounds : public VolumeBounds {
     eHalfLengthZ = 2,
     eHalfPhiSector = 3,
     eAveragePhi = 4,
+    eAngle = 5,
```

Comment:
* can we name this `eBevelAngle` ?

I think we need different angles on both sides, i.e.

`eBevelMinZ`
`eBevelMaxZ`

Because we will need different use cases for the ATLAS MS and the Athena-EIC beam pipe:

![Screenshot 2021-12-09 at 09 51 29](https://user-images.githubusercontent.com/26623879/145364380-822066bc-e248-4696-99da-175eca3985d8.png)

Reply:
Done!